src/components/Principle.svelte

```diff
@@ -42,6 +43,14 @@
 
 <div class={className}>
   <Header>{principle.num}: {principle.handle}</Header>
+  <label for={`principle-applicability-${principle.num}`}>Applicable?</label>
+  <input 
+    type="checkbox" 
+    id={`principle-applicability-${principle.num}`} 
+    bind:checked={$evaluation['meta']['principleApplicability'][principle.num]} 
```

Comment:
let Svelte automatically take care of the value of this checkbox, by pointing it directly to the right bit of the evaluation